### PR TITLE
[Checkout] Quick fix behat command on travis

### PR DESCRIPTION
**Pull request by Arminek**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| License       | MIT

### CODE REVIEW

File: etc/travis/run-behat [hunk 1]
```diff
@@ -9,11 +9,11 @@ if [[ $EXIT_CODE -eq 1 ]]; then
 fi;
 
 if [[ $TRAVIS_PULL_REQUEST = false && ! -z "$SYMFONY__PAYPAL__EXPRESS_CHECKOUT__USERNAME" ]]; then
-    bin/behat --strict -p cached --tags=paypal; EXIT_CODE=$?
+    bin/behat --strict -p cached --tags "paypal&&~todo"; EXIT_CODE=$?
 fi;
 
 if [[ $EXIT_CODE -eq 1 && $TRAVIS_PULL_REQUEST = false && ! -z "$SYMFONY__PAYPAL__EXPRESS_CHECKOUT__USERNAME" ]]; then
-    bin/behat --strict -p cached --rerun --tags=paypal; EXIT_CODE=$?
+    bin/behat --strict -p cached --rerun --tags "paypal&&~todo"; EXIT_CODE=$?
```
Comment (pamil) [inline etc/travis/run-behat]: Just my obsession, can we make a space before and after `&&`?

Comment (pamil) [inline etc/travis/run-behat]: And add `@` before tag names to be consistent with suites filters etc.

File: etc/travis/run-behat [hunk 2]
```diff
@@ -9,11 +9,11 @@ if [[ $EXIT_CODE -eq 1 ]]; then
 fi;
 
 if [[ $TRAVIS_PULL_REQUEST = false && ! -z "$SYMFONY__PAYPAL__EXPRESS_CHECKOUT__USERNAME" ]]; then
-    bin/behat --strict -p cached --tags=paypal; EXIT_CODE=$?
+    bin/behat --strict -p cached --tags "paypal&&~todo"; EXIT_CODE=$?
 fi;
 
 if [[ $EXIT_CODE -eq 1 && $TRAVIS_PULL_REQUEST = false && ! -z "$SYMFONY__PAYPAL__EXPRESS_CHECKOUT__USERNAME" ]]; then
```
Comment (pamil) [inline etc/travis/run-behat]: We may put it inside the earlier `if`.

**Comment by pamil** [comment]
There's a possibility that Behat fails whereas Travis passes.

Without PayPal:
1) bin/behat -> $EXIT_CODE = 1
2) bin/behat -> $EXIT_CODE = 1
Exit 1

With PayPal:
1) bin/behat -> $EXIT_CODE = 1
2) bin/behat -> $EXIT_CODE = 1
3) bin/behat --tags "paypal" -> $EXIT_CODE = 0 (PayPal suite passes)
Exit 0

We have to change:
```bash
bin/behat ; EXIT_CODE = $?
```

to:
```bash
bin/behat || EXIT_CODE = $?
```

File: etc/travis/run-behat (hunk 1 above)
Comment (pjedrzejewski) [inline etc/travis/run-behat]: :+1: for Kamil's obsession.

**Comment by Arminek** [comment]
It won't work anyway, but thanks for tips :+1:

File: etc/travis/run-behat [hunk 3]
```diff
@@ -1,19 +1,28 @@
...
 fi;
 
-if [[ $EXIT_CODE -eq 1 && $TRAVIS_PULL_REQUEST = false && ! -z "$SYMFONY__PAYPAL__EXPRESS_CHECKOUT__USERNAME" ]]; then
-    bin/behat --strict -p cached --rerun --tags=paypal; EXIT_CODE=$?
+if [[ $EXIT_BEHAT_PAYPAL_TEST_CODE -eq 1 && $TRAVIS_PULL_REQUEST = false && ! -z "$SYMFONY__PAYPAL__EXPRESS_CHECKOUT__USERNAME" ]]; then
+    bin/behat --strict -p cached --rerun --tags "@paypal && ~@todo"; EXIT_BEHAT_PAYPAL_TEST_CODE=$?
 fi;
 
-exit $EXIT_CODE
+checkBuild $EXIT_BEHAT_TEST_CODE $EXIT_BEHAT_PAYPAL_TEST_CODE
```
Comment (michalmarcinkowski) [inline etc/travis/run-behat]: The following will not work?
```
$EXIT_CODE = [ $EXIT_BEHAT_TEST_CODE -eq 0 ]] && [[ $EXIT_BEHAT_PAYPAL_TEST_CODE -eq 0 ]

exit $EXIT_CODE
```

**Comment by pjedrzejewski** [comment]
:+1:

**Pull request merged**